sql-db/reactive-vanilla/src/main/java/io/quarkus/qe/model/Book1.java

```diff
@@ -0,0 +1,49 @@
+package io.quarkus.qe.model;
+
+import java.util.List;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+import io.vertx.mutiny.pgclient.PgPool;
+import io.vertx.mutiny.sqlclient.Row;
+import io.vertx.mutiny.sqlclient.RowSet;
+import io.vertx.mutiny.sqlclient.Tuple;
+
+public class Book1 extends Book {
```

Comment:
Don't particularly like `Book1 extends Book` - can we have some other class names?

Like maybe `SoftCoverBook` and `HardCoverBook`? It's just that `Book1` vs `Book2` vs `Book` are too similar.

Reply:
Makes sense. It was originally named more diversely.